airflow_ai_sdk/models/tool.py

```diff
@@ -13,13 +13,15 @@ class WrappedTool(PydanticTool[AgentDepsT]):
     """
 
     async def run(
-        self, message: _messages.ToolCallPart, run_context: RunContext[AgentDepsT]
+        self,
+        message: _messages.ToolCallPart,
+        *args: any,
```

Comment:
nit: we should add kwargs too, just in case

```suggestion
        self,
        message: _messages.ToolCallPart,
        *args: Any,
        **kwargs: Any,
```

Reply:
will need to also add `from typing import Any` to the top - `any` is an object, not a type